packages/react-heartwood-components/src/components/Core/components/HeaderPrimary/HeaderPrimary.js

```diff
@@ -135,7 +135,7 @@ export default class HeaderPrimary extends Component<Props, State> {
 							)}
 							<UserMenu
 								menuIsVisible={isUserMenuVisible}
-								toggleMenu={this.toggleUserMenuVisibility}
+								setMenu={this.setUserMenuVisibility}
```

Comment:
Did you add this prop to `UserMenu` or was it already there?

Reply:
I didn't add it but I did change the name....should this one have been left as `toggle`?

Reply:
I just didn't see the code here where you made that change. This is fine if that's checked in already.